### PR TITLE
Add libpng 1.6.41

### PR DESCRIPTION
**Pull request by Vertexwahn**
Add libpng 1.6.41

### CODE REVIEW

**Comment by Vertexwahn** [comment]
Need `skip-url-stability-check` label. (Switched to GitHub source archive, since Sourceforge sucks)

File: modules/libpng/1.6.41/presubmit.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - macos
+  - macos_arm64
+  - ubuntu2004
+  - ubuntu2204
+  - windows
+  bazel: [6.x, 7.x]
+tasks:
+  verify_targets:
+    name: Verify build targets
+    platform: ${{ platform }}
```
Comment (fmeum) [inline modules/libpng/1.6.41/presubmit.yml:15]:
```suggestion
    platform: ${{ platform }}
    bazel: ${{ bazel }}
```

**Review by fmeum — APPROVED** [review]

File: modules/libpng/1.6.41/presubmit.yml (hunk 1 above)
Comment (fmeum) [reply modules/libpng/1.6.41/presubmit.yml:15]: Sorry, please remove the duplicated line.

Comment (Vertexwahn) [reply modules/libpng/1.6.41/presubmit.yml:15]: done

**Pull request merged**